admin/partials/cover_art.php

```diff
@@ -0,0 +1,81 @@
+<p class="description">
+	<?php printf(
+		wp_kses(
+			__( 'You can set one or more <a href="%s">cover art</a> images below.', 'apple-news' ),
+			array( 'a' => array( 'href' => array() ) )
+		),
+		esc_url( __( 'https://developer.apple.com/library/content/documentation/General/Conceptual/Apple_News_Format_Ref/CoverArt.html', 'apple-news' ) )
```

Comment:
No need to translate a URL and since this is hardcoded, it doesn't need escaping either